link

```diff
@@ -37,7 +37,12 @@ if (!is_dir("$argv[1]/vendor/symfony")) {
 $sfPackages = array('symfony/symfony' => __DIR__);
 
 $filesystem = new Filesystem();
-foreach (glob(__DIR__.'/src/Symfony/{Bundle,Bridge,Component,Component/Security}/*', GLOB_BRACE | GLOB_ONLYDIR | GLOB_NOSORT) as $dir) {
+$braces = ['Bundle', 'Bridge', 'Component', 'Component/Security'];
+$directories = call_user_func_array('array_merge', array_values(array_map(function ($part) {
```

Comment:
On the `4.0` branch and onwards (where PHP 5.6 support has been dropped) this line could be rewritten as:

```php
$directories = array_merge(...array_values(array_map(function (string $part): array {
```